src/Engine/Utils.java

```diff
@@ -92,140 +92,136 @@ public static void highlightLocations(GameMap map, ArrayList<XYCoord> locations)
     }
   }
 
+  /** Alias for {@link #findPossibleDestinations(XYCoord, Unit, GameMap, boolean) findPossibleDestinations()} **/
+  public static ArrayList<XYCoord> findPossibleDestinations(Unit unit, GameMap gameMap)
+  {
+    return findPossibleDestinations(new XYCoord(unit.x, unit.y), unit, gameMap);
+  }
+  /** Alias for {@link #findPossibleDestinations(XYCoord, Unit, GameMap, boolean) findPossibleDestinations()} **/
+  public static ArrayList<XYCoord> findPossibleDestinations(XYCoord start, Unit unit, GameMap gameMap)
+  {
+    return findFloodFillArea(start, unit.getMoveFunctor(), Math.min(unit.model.movePower, unit.fuel), gameMap);
+  }
   /**
    * Returns the list of XYCoords in gameMap reachable by unit this turn.
    * @param unit The unit to evaluate.
    * @param gameMap The map to search over.
    * @param includeOccupiedSpaces If true, will include spaces occupied by a friendly unit, if some action could end on this space (e.g. LOAD, JOIN).
    */
-  public static ArrayList<XYCoord> findPossibleDestinations(Unit unit, GameMap gameMap, boolean includeOccupiedSpaces)
+  public static ArrayList<XYCoord> findFloodFillArea(XYCoord start, FloodFillFunctor fff, int initialFillPower, GameMap gameMap)
   {
     ArrayList<XYCoord> reachableTiles = new ArrayList<XYCoord>();
 
-    if( null == unit || unit.x < 0 || unit.y < 0 )
+    if( null == fff || null == start || start.xCoord < 0 || start.yCoord < 0 )
     {
       System.out.println("WARNING! Finding destinations for ineligible unit!");
       return reachableTiles;
     }
 
-    // set all locations to false/remaining move = 0
-    int[][] costGrid = new int[gameMap.mapWidth][gameMap.mapHeight];
+    // set all locations to unreachable/remaining move = 0
```

Comment:
Nothing is being set equal to zero here (except `i` and `i`, pedant).

Reply:
It's an unnecessary clarification to "unreachable", i.e. "You don't have any move here"